local-setup/jupyter/crs_loader.py

```
@@ -0,0 +1,1363 @@
+        try:
+            self._update_citymodule_db(module_code, tenant_code, module_config)
+        except Exception as e:
+            print(f"   ⚠️  {module_code}: Add tenant manually to citymodule (MDMS v2 API limitation)")
+
+    def _update_citymodule_db(self, module_code: str, tenant_code: str, current_config: dict):
+        """Update citymodule via database query through boundary-service workaround"""
+        import subprocess
+        import json as json_lib
+
+        # Add tenant to config
+        new_tenants = current_config.get("tenants", []) + [{"code": tenant_code}]
```

Ruff (0.15.0)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)